l10n-ms/pom.xml

```diff
@@ -94,6 +106,11 @@
 			<artifactId>logback-classic</artifactId>
 			<version>${logback.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>org.xmlunit</groupId>
+			<artifactId>xmlunit-core</artifactId>
+		</dependency>
```

Comment:
We have to put specific versions. Also comments about every dependency that is added.